net.lutris.Lutris.yml

```diff
@@ -407,8 +407,8 @@ modules:
     sources: &wine_sources
       - type: git
         url: "https://github.com/lutris/wine.git"
-        branch: lutris-4.16
-        commit: d67da44865e4818e6e477a70e7e85dc05186bf96
+        branch: lutris-ge-4.17
```

Comment:
Why `-ge-` indeed? Is it the new default for lutris wine?

Reply:
Not know. @strycore?

Reply:
No, we don't have a build with such name. Also, wine versions shouldn't be hardcoded anywhere.

Reply:
@tannisroot We are bundling Lutris Wine here as a "system" one, just like distributions provide vanilla or staging Wine.